wdl/finemap_inputs.json

```diff
@@ -1,9 +1,9 @@
 {
     "finemap.zones": "europe-west1-b europe-west1-c europe-west1-d",
-    "finemap.docker": "eu.gcr.io/finngen-refinery-dev/finemap-suite:dfad9e94ede4",
-    "finemap.sumstats_pattern": "gs://finngen-production-library-green/finngen_R6/finngen_R6_analysis_data/summary_stats/release/finngen_R6_{PHENO}.gz",
-    "finemap.phenolistfile": "gs://r6_data/finemap/conf/large_region_phenos",
-    "finemap.phenotypes": "gs://r6_data/pheno/finngen_R6_cov_pheno_V1.txt.gz",
+    "finemap.docker": "eu.gcr.io/finngen-refinery-dev/finemap-suite:ed26d2c",
+    "finemap.sumstats_pattern": "gs://r7_data/regenie/summary_stats_raw_release/{PHENO}.gz",
```

Comment:
Just realized that info filtered sumstats are actually here, not that it makes a big difference but better have this location as someone will eventually use wrong location in production runs:

gs://finngen-production-library-green/finngen_R7/finngen_R7_analysis_data/summary_stats/release/finngen_R7_{PHENO}gz

Reply:
good to know! addressed